unstructured/ingest/runner/writers/__init__.py

```diff
@@ -1,15 +1,15 @@
 import typing as t
 
-from .azure import AzureWriter
 from .azure_cognitive_search import AzureCognitiveSearchWriter
 from .base_writer import Writer
-from .box import BoxWriter
 from .delta_table import DeltaTableWriter
-from .dropbox import DropboxWriter
-from .gcs import GcsWriter
+from .fsspec.azure import AzureWriter
+from .fsspec.box import BoxWriter
+from .fsspec.dropbox import DropboxWriter
+from .fsspec.gcs import GcsWriter
+from .fsspec.s3 import S3Writer
```

Comment:
I don't see sftp in here. Just something to note. Maybe I need to do a ticket for it?